Week 2 P1 solution

lordoj opened the pull request:
prompt #1

https://github.com/Blue-movement/bootcamp-2020.1/commit/ad9bee6d07976f3aaaf92f339c49ed4f1bc34ec8

lordoj committed:
Create Oj-BademosiP1.py

prompt #1

barnesdc reviewed (changes requested):
Request you update the file names to follow the naming convention in the README.md.

lordoj committed:
Rename Oj-BademosiP1.py to Oj-Bademosi.py

lordoj committed:
Create OJ-Bademosi.py

Using a recursive function, print leaf to root path for every leaf node of the binary tree.

lordoj committed:
Problem 1 Week 8
